web-gateway/app/controllers/ItemForm.java

```diff
@@ -31,30 +31,21 @@
         List<ValidationError> errors = new ArrayList<>();
         
         try {
-            Currency c = Currency.valueOf(currency);
-
+        	Currency c = Currency.valueOf(currency);
             if (!c.isValidStep(increment.doubleValue())) {
                 errors.add(new ValidationError("increment", "invalid.step"));
             }
 
             if (!c.isValidStep(reserve.doubleValue())) {
                 errors.add(new ValidationError("reserve", "invalid.step"));
             }
-        } catch (IllegalArgumentException e) {
-            errors.add(new ValidationError("currency", "invalid.currency"));
-        }
-
-        // Make sure that the increment and reserve are multiples of 50c - in a real app, this would be more complex
-        // and based on currency specific rules, for now we'll assume currencies that have cents.
-        try {
-            int incrementInt = increment.multiply(BigDecimal.valueOf(2)).intValueExact();
-            if (incrementInt <= 0) {
-                errors.add(new ValidationError("increment", "invalid.increment"));
-            } else if (incrementInt >= 100) {
+            // Make sure that the increment and reserve are multiples of 50c - in a real app, this would be more complex
+            // and based on currency specific rules, for now we'll assume currencies that have cents.
```

Comment:
I think this comment is no longer accurate. With this change, it's not assuming cents, and the rule is currency-specific (though still fairly simple). I think the comment can just be deleted.